cob_twist_controller/scripts/test/test_careobot_st_jla_ca_sphere.py

```diff
@@ -91,20 +91,21 @@ def init_pos():
 if __name__ == "__main__":
     rospy.init_node("test_careobot_st_jla_ca_sphere")
 
-    base_dir = '/home/fxm-mb/bag-files/2015_08_06/'
+    base_dir = '/home/bbrito/bag-files/2016_02_22/'
     if rospy.has_param('~base_dir'):
         base_dir = rospy.get_param('~base_dir')
     else:
         rospy.logwarn('Could not find parameter ~base_dir. Using default base_dir: ' + base_dir)
 
-    if rospy.has_param('chain_tip_link'):
+    action_name = rospy.get_namespace()
+    if rospy.has_param(action_name+'chain_tip_link'):
         chain_tip_link = rospy.get_param('chain_tip_link')
     else:
         rospy.logwarn('Could not find parameter chain_tip_link.')
         exit(-1)
 
-    if rospy.has_param('frame_tracker/tracking_frame'):
-        tracking_frame = rospy.get_param('frame_tracker/tracking_frame')
+    if rospy.has_param(action_name+'frame_tracker/target_frame'):
+        tracking_frame = rospy.get_param(action_name+'frame_tracker/target_frame')
```

Comment:
if you run this node like `rosrun cob_twist_controller test_careobot_st_jla_ca_sphere.py __ns:=arm_left`, you do not need these changes...